benchmarks/longmemeval_v2_chunk_geometry/stage0b.py

```
@@ -0,0 +1,223 @@
+        pieces = slices_for(entry)
+        if len(lines) < 2 or not pieces:
+            continue
+        owner = {}
+        for index, piece in enumerate(pieces):
+            for line_index in piece["line_indices"]:
+                owner[line_index] = index
+        for k in SPAN_SIZES:
+            if len(lines) <= k:
+                continue
+            for _ in range(max(1, SPAN_SAMPLES // len(sampled))):
+                start = random.randrange(0, len(lines) - k)
```

Ruff (0.15.21)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)